ext/configuration.h

```diff
@@ -84,7 +84,7 @@ enum ddtrace_dbm_propagation_mode {
     CONFIG(BOOL, DD_TRACE_AUTO_FLUSH_ENABLED, "false")                                                         \
     CONFIG(BOOL, DD_TRACE_CLI_ENABLED, "false")                                                                \
     CONFIG(BOOL, DD_TRACE_MEASURE_COMPILE_TIME, "true")                                                        \
-    CONFIG(BOOL, DD_TRACE_DEBUG, "false")                                                                      \
+    CONFIG(SET, DD_TRACE_DEBUG, "Error", .ini_change = ddtrace_alter_dd_trace_debug)                           \
```

Comment:
`DD_TRACE_DEBUG` is a bool in all languages https://docs.datadoghq.com/tracing/troubleshooting/tracer_debug_logs/?code-lang=java#enable-debug-mode. And I don't think we have a log level one, by default we log everything starting at Info, unless `DD_TRACE_DEBUG` is true (at least in .NET)

Reply:
Note that boolean functionality is retained in log.rs; if the first arg is 1, On or true, it will have the same behaviour as before.